test/verify/check-dashboard

```diff
@@ -70,17 +70,17 @@ class TestDashboard(MachineCase):
             b.click('#dashboard_setup_server_dialog .btn-primary')
         b.wait_popdown('dashboard_setup_server_dialog')
 
+class TestBasicDashboard(MachineCase, DashBoardHelpers):
```

Comment:
> I would suggest having a class variable here called "machines", that we can just set right here instead of having to override setUp().

I called it `additional_machines`, `machines` was already used